server/mcp_server_cloudidentity/README.md

```diff
@@ -0,0 +1,197 @@
+# cloudidentity MCP Server
+
+| 版本 |         v1.0.0          |
+|:--: |:-----------------------:|
+| 描述 | 云身份中心（Cloud Identity）是链接外部身份和火山引擎云上身份的统一管理平台。使用云身份中心，您可以统一管理您的企业员工登录火山引擎的方式、一键同步员工身份至火山引擎，并可以基于企业组织的多账号管理，一次性配置企业身份管理系统到火山引擎多账号下的单点登录、及时对企业员工“入离转调”产生的访问权限变动进行管理和同步。       |
+| 分类 | 工具类 |
+| 标签 | 管理、云身份、用户、角色、管理与治理           |
+
+## Tools
+本 MCP Server 产品提供以下 Tools:
+
+
+- AttachPolicyToUserProvision: 为用户访问授权任务附加系统预设策略
```

Comment:
disable this modify method and allow user to enable
AttachPolicyToUserProvision